site/pt/blog/new-in-devtools-111/index.md

```diff
@@ -113,39 +135,50 @@ Chromium issues: [1407586](https://crbug.com/1407586), [1402891](https://crbug.c
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q5ZeDvMc3OseF8fQve5A.png", alt="Caches in the Application panel.", width="800", height="506" %}
+O painel **Cache Storage** agora pode ser encontrado na seção **Armazenamento** do painel **Aplicação**, enquanto o painel **Back/forward cache** foi movido para a seção **Background Services**.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q5ZeDvMc3OseF8fQve5A.png", alt="Caches no painel de aplicação.", width="800", height="506" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/adccf1afe5d585b294dee247f5a4982aca8f5f1e  #}
 
-Chromium issue: [1407166](https://crbug.com/1407166)
+Issue relacionada: [1407166](https://crbug.com/1407166)
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Outras novidades {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Mais algumas novidades dignas de nota nessa release:
 
 <!-- - DevTools has been updated to respect the [Disable cache](/docs/devtools/network/reference/#disable-cache) setting when loading sourcemaps. ([1407084](https://crbug.com/1407084)) -->
+- O DevTools foi atualizado para respeitar a configuração [Desativar cache](/docs/devtools/network/reference/#disable-cache) ao carregar sourcemaps. ([1407084](https://crbug.com/1407084))
 <!-- - The **Elements** panel now instantly autofocuses on the first matching element in search results. ([1381853](https://crbug.com/1381853)) -->
+- O painel **Elementos** agora foca automaticamente no primeiro elemento correspondente nos resultados da pesquisa. ([1381853](https://crbug.com/1381853))
 <!-- - Various fixes to improve the sourcemap and breakpoints reliability. ([508270](https://crbug.com/508270), [1403362](https://crbug.com/1403362), [1403432](https://crbug.com/1403432), [1396298](https://crbug.com/1396298), [1395337](https://crbug.com/1395337), [1405134](https://crbug.com/1405134)) -->
+- Várias correções para melhorar a confiabilidade de sourcemaps e dos breakpoints. ([508270](https://crbug.com/508270), [1403362](https://crbug.com/1403362), [1403432](https://crbug.com/1403432), [1396298]( https://crbug.com/1396298), [1395337](https://crbug.com/1395337), [1405134](https://crbug.com/1405134))
 <!-- - To better facilitate debugging, DevTools now supports evaluating expressions with private class members. ([1381806](https://crbug.com/1381806)) -->
-    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q68fvo870weBVwupujqf.png", alt="Evaluating expressions with private class members.", width="800", height="683" %}
+- Para facilitar melhor o debugging, o DevTools agora oferece suporte à avaliação de expressões com membros de classes privadas. ([1381806](https://crbug.com/1381806))
+     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q68fvo870weBVwupujqf.png", alt="Evaluating expressions with private class members.", width="800", height="683" %}
```

Comment:
the alt here is in English.

Reply:
"Avaliando expressões com membros de classe privados."

Reply:
```suggestion
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/q68fvo870weBVwupujqf.png", alt="Avaliando expressões com membros de classe privados.", width="800", height="683" %}
```